src/view/UserphraseView.h

```diff
@@ -64,4 +64,7 @@ public slots:
     UserphraseViewMenu *menu_;
 
     bool dialogType_;
+
+    std::shared_ptr<QString> originalPhrase;
```

Comment:
No, because the interface in `UserphraseSortFilterProxyModel` 
need `std::shared_ptr<QString>`, and I think I should keep the consistency 
of the previous function call of model()->add()